contents/blog/the-posthog-array-1-37-0.mdx

```diff
@@ -0,0 +1,145 @@
+---
+date: 2022-06-27
+title: 'Array 1.37.0: Cohorts 2.0 and dancing hedgehogs'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.37.0 introduces advanced filtering for cohorts, improved events and property pages, dancing hedgehogs and much, much more.
+---
+
+import Cohorts2 from '../images/blog/array/1-37-0-cohorts-2-0.png'
+import DancingHog from '../images/blog/array/1-37-0-dancing-hog.gif'
+import EventPage from '../images/blog/array/1-37-0-event-page.png'
+import ConfigurableColumns from '../images/blog/array/1-37-0-configurable-columns.png'
+
+Want to know more about what we're up to? [Subscribe to HogMail, our new newsletter](/newsletter), which we send once every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/self-host/configure/upgrading-posthog).
+
+## PostHog 1.37.0 release notes
+
+**Release highlights:**
+
+- New: [Cohorts 2.0](#new-and-or-filtering)
+- New: [Dancing hedgehogs](#new-multi-dashboard-insights)
+- New: [Event and property detail pages](#improved-a-more-accessible-color-palette)
+- Improved: [Save event columns](#improved-a-more-accessible-color-palette)
+- Improved: [Faster response times](#improved-series-switching-in-persons)
+
+### New: Cohorts 2.0
```

Comment:
@EDsCODE can you give this section on cohorts a read and change anything that you think should be updated.